Tests/IO.lean

```diff
@@ -36,14 +36,11 @@ def main := do
     parseT (string "ab" *> string "cd" <* eol <* eof : PIO String) ("", h')
 
   lspecIO $
-    withExceptOk "file, manual newline: parsing successful"
-      (Either.either .error .ok fIO)
+    withExceptOk "file, manual newline: parsing successful" fIO
```

Comment:
Took me a while to understand that you used to have an adapter from either into except. 😆 Oops